runtime/list.go

```diff
@@ -334,10 +334,20 @@ func listSetItem(f *Frame, o, key, value *Object) *BaseException {
 	return f.RaiseType(TypeErrorType, fmt.Sprintf("list indices must be integers, not %s", key.Type().Name()))
 }
 
+func listSort(f *Frame, args Args, _ KWArgs) (*Object, *BaseException) {
```

Comment:
As mentioned in the sorted() PR, list.sort() is a little more complicated than this:

`list.sort(cmp=None, key=None, reverse=False)`

A TODO is fine for now.